doc/docs.md

```diff
@@ -6191,6 +6191,7 @@ To improve safety and maintainability, operator overloading is limited.
 
 - When overriding `<` and `==`, the return type must be strictly `bool`.
 - Both arguments must have the same type (just like with all operators in V).
+- Overloaded operators has to return the same type as the argument (with the expection `<` and `==` which returns boolean expressions).
```

Comment:
As this line goes beyond the 100 words. I think

```suggestion
- Overloaded operators have to return the same type as the argument (with the exception `<` and `==).
```

works better

Reply:
Wow, trying to write to fast hehe :/